source/_docs/unsupported-modules-plugins.md

```diff
@@ -220,9 +222,14 @@ $conf[‘schema_suppress_type_warnings’] = TRUE;
 ```
 
 <hr>
-### [Taxonomy CSV](https://www.drupal.org/project/taxonomy_csv)  
+### [Taxonomy CSV](https://www.drupal.org/project/taxonomy_csv)
 **Issue**:  This module requires the use of the `/tmp` directory. See [Using the tmp Directory](/docs/unsupported-modules-plugins/#using-the-tmp-directory) section below.
 <hr>
+
+### [Twig Extensions](https://www.drupal.org/project/twig_extensions)  
+**Issue**:  This module uses `php-intl`, which is not currently supported by Pantheon.
```

Comment:
Can we get a add a link for`php-intl` for additional context on what's not supported?

[This issue](https://www.drupal.org/node/2694185) indicates there may be a solution, can you test and validate that potential fix @ilovelamp?